### ISSUE TITLE
BUG: plays and playtime are doubled when using the --aside option

### ISSUE THREAD

**Issue by emanuele6** [ishwi/Chuu#52]
Example:
```
!timeartists a 3x3 --plays
```
<details>
  <summary>output</summary>

![image](https://user-images.githubusercontent.com/20175435/123348131-8085f380-d559-11eb-811b-41c0426617c6.png)
</details>

```
!timeartists a 3x3 --plays --aside
```
<details>
  <summary>output</summary>

![image](https://user-images.githubusercontent.com/20175435/123348410-909dd300-d559-11eb-987d-c31ac19076dc.png)
</details>

**Comment by emanuele6** [comment]
I just noticed this also happens with the `!chart` command and, I assume, any command that has `--aside`.

**Comment by ishwi** [comment]
Should be fixed with the changes introduced in 15070de6fea2a648e06d8b06ec7af0b74ab24acb

Idek how this one went unnoticed for so long and tyvm for reporting it! ❤️
